lib/bolt/outputter/human.rb

```diff
@@ -43,9 +43,9 @@ def start_spin
       end
 
       def stop_spin
-        return unless @spin && @stream.isatty
+        return unless @spin && @stream.isatty && @spinning
```

Comment:
```suggestion
        return unless @spin && @stream.isatty && @spinning
```

There's something wonky happening with checking the state of the spinner and starting/stopping it when using the `parallelize` function. Running a plan that calls `parallelize`, followed by the `prompt` function, results in the spinner showing for a prompt. Here's an example plan that shows this behavior:

```puppet
plan bolt (
  TargetSpec $targets = 'localhost',
) {
  $_targets = get_targets($targets)

  $results = parallelize($_targets) |$target| {
    out::message('hello')
  }

  $prompt = prompt('prompt')

  return $results
}
```

Removing `@spinning` from the conditional fixes it, so I suspect that's just not being set correctly when calling `parallelize`?

Reply:
Hmmm, I'm not seeing that...maybe we can talk about it at standup tomorrow? The `@spinning` needs to be there so that multiple spinners aren't printed when in a parallelize block.